.github/workflows/continuous-deployment.yml

```diff
@@ -469,18 +479,12 @@ jobs:
           -g ${{ steps.rgApp.outputs.out }} \
           -n ${{ needs.build-info.outputs.apiName }} \
           --src ./${{ needs.build-info.outputs.apiName }}.zip \
-          --settings "MSSQL_HOST=${{ secrets.AZ_MSSQL_HOST }} \
-          MSSQL_DATABASE=${{ secrets.AZ_MSSQL_DATABASE }} \
-          MSSQL_DATABASE_DXTR=${{ secrets.AZ_MSSQL_DATABASE_DXTR }} \
-          MSSQL_USER=${{ secrets.AZ_MSSQL_USER }} \
-          MSSQL_PASS=${{ secrets.AZ_MSSQL_PASS }} \
-          MSSQL_ENCRYPT=${{ secrets.AZ_MSSQL_ENCRYPT }} \
-          MSSQL_TRUST_UNSIGNED_CERT=${{ secrets.AZ_MSSQL_TRUST_UNSIGNED_CERT }} \
-          STARTING_MONTH=${{ vars.STARTING_MONTH }} \
+          --kvName ${{ secrets.AZ_KV_APP_CONFIG_NAME }} \
+          --kvSettings "MSSQL_HOST MSSQL_DATABASE_DXTR MSSQL_USER MSSQL_PASS MSSQL_ENCRYPT MSSQL_TRUST_UNSIGNED_CERT FEATURE_FLAG_SDK_KEY" \
+          --settings "STARTING_MONTH=${{ vars.STARTING_MONTH }} \
```

Comment:
Do we want to move `STARTING_MONTH` into keyvault?

Reply:
Nope, makes sense in LD or as a normal env var